flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_MCDU_KB.js

```diff
@@ -0,0 +1,75 @@
+class CDU_OPTIONS_MCDU_KB {
+    static ShowPage(mcdu) {
+
+        mcdu.clearDisplay();
+
+        const mcduInput = NXDataStore.get("MCDU_KB_INPUT", "DISABLED");
```

Comment:
same here. `mcdu.isInteractive()`

And when doing this, I'd rename (for simplicity) `mcduInput` into `mcduIsInteractive` so the variable name actually represents it's functionality/usage.

Reply:
See above comment on get isInteractive()